### PR TITLE
shell scripts fail on first error

### PR DESCRIPTION
**Pull request by guillaumerose**
The script now properly fails in case of an error in a command.

### CODE REVIEW

File: createdisk.sh [hunk 1]
```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
+set -exuo pipefail
+
```
Comment (cfergeau) [inline createdisk.sh:4]: iirc this was causing some unexpected failures in the past, I fixed some of them, but I forgot which state we are in. Could you test this, or should we wait for CI?

Comment (guillaumerose) [reply createdisk.sh:4]: You caught me, I was planning to do CI Driven Development :)

File: createdisk.sh [hunk 2]
```diff
@@ -325,7 +325,7 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo timedatectl set-ntp off
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo systemctl enable io.podman.socket
 
 # Remove all the pods from the VM
-${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo crictl stopp $(sudo crictl pods -q)'
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo crictl stop $(sudo crictl pods -q)'
```
Comment (praveenkumar) [inline createdisk.sh:328]: @guillaumerose it should be `stopp` for pods.

Comment (guillaumerose) [reply createdisk.sh]: ah thanks! I thought it was a typo

**Comment by guillaumerose** [comment]
All green! Ready to be reviewed.

File: createdisk.sh [hunk 3]
```diff
@@ -341,11 +343,12 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo systemctl enable io.podman
 
 # Remove all the pods except openshift-sdn from the VM
 pods=$(${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo crictl pods -o json | jq '.items[] | select(.metadata.namespace != "openshift-sdn")' | jq -r .id)
-${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo crictl stopp ${pods}
-${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- "for i in {1..3}; do sudo crictl rmp "${pods}" && break || sleep 2; done"
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- "sudo crictl stopp ${pods} || true"
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- "for i in {1..3}; do sudo crictl rmp "${pods}" && break || sleep 2; done || true"
```
Comment (praveenkumar) [inline createdisk.sh:347]: :+1:

**Review by praveenkumar — APPROVED** [review]

**Pull request merged**